contracts/starknet/lib/voting.cairo

```diff
@@ -505,20 +497,16 @@ namespace Voting {
         );
 
         let (no_voting_power) = uint256_eq(Uint256(0, 0), user_voting_power);
-
         with_attr error_message("Voting: No voting power for user") {
             assert no_voting_power = 0;
         }
 
         let (previous_voting_power) = Voting_vote_power_store.read(proposal_id, choice);
-        let (new_voting_power, overflow) = uint256_add(user_voting_power, previous_voting_power);
-
         with_attr error_message("Voting: Overflow in voting power") {
```

Comment:
Can we nest `error_message`s ?
Because the library already has an error message: https://github.com/OpenZeppelin/cairo-contracts/blob/893f4b98739bdc1b7d64d0511d96d19f290eef88/src/openzeppelin/security/safemath/library.cairo#L29-L31

Reply:
yeah you can nest them. the outer one isnt needed It just makes it easier to see where the error occurs i thought.